cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/config/WebserverUrlPrefixConfigTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.config;
+
+import com.linkedin.kafka.cruisecontrol.config.constants.WebServerConfig;
+import org.apache.kafka.common.config.ConfigException;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.easymock.EasyMock.*;
+
+
+public class WebserverUrlPrefixConfigTest {
+
+  @Rule
+  public ExpectedException _expectedException = ExpectedException.none();
+
+  @Test
+  public void testWebserverUrlPrefixConfigsCorrectSetup() {
+    KafkaCruiseControlConfig config = partialMockBuilder(KafkaCruiseControlConfig.class)
+        .addMockedMethod("getString")
+        .createNiceMock();
+
+    expect(config.getString(WebServerConfig.WEBSERVER_API_URLPREFIX_CONFIG)).andReturn("a/b/c/d/*");
+    expect(config.getString(WebServerConfig.WEBSERVER_UI_URLPREFIX_CONFIG)).andReturn("a/b/c/*");
+    replay(config);
+    config.sanityCheckWebServerUrlPrefix();
+    verify(config);
+  }
+
+  @Test
+  public void testWebserverUrlPrefixConfigCorrectSetupOnDefault() {
```

Comment:
I think the value of this test is to ensure that if those default values ever change to something that is not valid, this test will catch it

Reply:
Don't we have sanity checks in `KafkaCruiseControlConfig.java` to ensure validity of config values in production regardless of whether they are the default ones or overridden by users?

Reply:
Yes, the sanity checks will catch invalid web server URL prefix. However, invalid URL prefix can only be caught during runtime whereas this test fails the build process if the default webserver URL is invalid.

Reply:
The existing unit tests already verify (as they should) the behavior of the tested unit with all possible good and bad inputs. The default value is just one such input, whose behavior is known -- because we already tested what such an input would generate. We know that if a future programmer changes the default value, CC will fail to start and provide a meaningful message to help resolve the configuration issue, because it is tested.

I am against adding individual tests specifically for the default values as it eventually leads to bloating, making a build take longer at CircleCI with no additional testing coverage.

Reply:
Makes sense. I'll remove it